single_include/indicators/indicators.hpp

```diff
@@ -2089,7 +2089,9 @@ class BlockProgressBar {
 
       if (saved_start_time) {
         auto eta = std::chrono::nanoseconds(
-            progress_ > 0 ? static_cast<long long>(elapsed.count() * max_progress / progress_) : 0);
+            progress_ > 0 ? static_cast<long long>(elapsed.count() *
+                                                   (max_progress / progress_))
+                          : 0);
```

Comment:
Since the single_include is generated, can you make these changes to `include/indicators/block_progress_bar.hpp` and `include/indicators/progress_spinner.hpp`?

Reply:
Can do. Is the appropriate call for running the generator documented somewhere so I can make sure single include is updated the correct way as well?

Reply:
Yes! Here: https://github.com/p-ranav/indicators#generating-single-header

Reply:
I looked right over it! Thanks!